package.json

```diff
@@ -7,7 +7,9 @@
     "test": "test"
   },
   "scripts": {
-    "test": "./test/runtests.sh"
+    "test": "./test/runtests.sh",
+    "compile": "rm -rf lib/ && babel --source-maps -d lib/ src/",
```

Comment:
Let's put --source-maps in .babelrc

Or even better let's see if we can't make a single npm module containing babel config and linter settings... 
Probably outside the scope of this PR.